src/cpu/core_dyn_x86/cache.h

```diff
@@ -240,7 +246,9 @@ class CodePageHandler : public PageHandler {
 		}
 		addr&=4095;
 		if (host_readd(hostmem+addr)==(Bit32u)val) return false;
-		if (!*(Bit32u*)&write_map[addr]) {
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the `host_*` functions do).